system/jlib/jlog.cpp

```diff
@@ -2863,25 +2868,33 @@ class DummyLogCtx : implements IContextLogger
     {
         return 0;
     }
-    virtual void setGlobalId(const char *id, SocketEndpoint &ep, unsigned pid) override
+    virtual void setActiveSpan(ISpan * span) override
```

Comment:
minor, but this overloads the OTel "active span" concept, but it doesn't seems to convey a slightly different meaning. I would consider another term (not sure what it would be)

Reply:
> From what I've seen of the OTel active span concept it's basically for single threaded applications. I see this as the "per query" version of the active span concept.

@afishbeck is this in reply to my comment to Gavin?
If so, it is still overloading the term and can be misunderstood. It should go without saying, and I'm sure you'll agree, we should avoid any/all potential confusion if possible.

Reply:
Right, I could see someone already familiar with the OTel API making an assumption about what this means when they read the name.

Reply:
Any suggestions for a better name?  setCurrentSpan()?

Reply:
current span was my immediate thought as well.